modules/040-node-manager/images/early-oom/src/psi-monitor.go

```diff
@@ -35,16 +40,42 @@ const (
 	sysrqOOMCharacter = "f"
 )
 
+var psiAvailable = prometheus.NewGauge(prometheus.GaugeOpts{
+	Name: "early_oom_psi_unavailable",
+	Help: "Whether PSI subsystem is unavailable on a given system",
+})
+
+func init() {
+	prometheus.MustRegister(psiAvailable)
+}
+
 func main() {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, syscall.SIGTERM, syscall.SIGINT)
 
+	server := &http.Server{Addr: "127.0.0.1:8080"}
+	http.Handle("/metrics", promhttp.Handler())
+
+	go func() {
+		if err := server.ListenAndServe(); err != nil {
+			log.Fatal(err)
+		}
+	}()
+
+	psiSupportErr := probePSISupport()
+	if psiSupportErr != nil {
+		log.Println(psiSupportErr)
+		psiAvailable.Set(1)
```

Comment:
Because this one is reading in an opposite meaning

Reply:
Fixed.